core/src/main/groovy/noe/rhel/server/workspace/WorkspaceHttpdTomcatsBaseOS.groovy

```diff
@@ -46,22 +47,51 @@ class WorkspaceHttpdTomcatsBaseOS extends WorkspaceMultipleTomcats{
     httpd.updateConfSetBindAddress(hostIpAddress)
 
     copyModulesIfMissing(httpd)
+    copyConfIfMissing(httpd)
     workspaceTomcat.prepare(true) //always true due to running with JWS 5 `ewsVersion >= new Version('3.1.0-DR0')`
 
     createAdditionalTomcatsWithRegistrations(numberOfAdditionalTomcats)
     originalTomcatHosts.putAll(originalTomcatHostsIpAddresses())
   }
 
+  /**
+   * JBCS modules are provided in different folder and need to be properly loaded or copied for smooth working.
+   * @param httpd Instance of `Httpd` server
+   */
   private void copyModulesIfMissing(Httpd httpd) {
-    File modulePath = new File("${httpd.getServerRoot()}/modules/mod_jk.so")
-    File sclModulePath = new File("${DefaultProperties.HTTPD_SCL_ROOT}/usr/lib64/httpd/modules/mod_jk.so")
-    if (!modulePath.exists() && sclModulePath.exists()) {
-      log.info("Copiing file ${sclModulePath} to modules folder of httpd ${modulePath}.")
-      modJkModules.add(modulePath)
-      JBFile.copyFile(sclModulePath, modulePath)
+    List<String> moduleNames = ["mod_jk", "mod_advertise", "mod_cluster_slotmem", "mod_manager", "mod_proxy_cluster"]
+    moduleNames.each() { String moduleName ->
+      File modulePath = new File("${httpd.getServerRoot()}/modules/${moduleName}.so")
+      File sclModulePath = new File("${DefaultProperties.HTTPD_SCL_ROOT}/usr/lib64/httpd/modules/${moduleName}.so")
+      if (!modulePath.exists() && sclModulePath.exists()) {
+        log.info("Copying file ${sclModulePath} to modules folder of httpd ${modulePath}.")
```

Comment:
logging...

Reply:
`JBFile.copyFile()` doesn't contain loggin at all.

Reply:
ok, if it does not, is it neccessary to print it to info level?

Reply:
Right, debug logging level is expected present. Updated